driver-kotlin-sync/src/main/kotlin/com/mongodb/kotlin/client/MongoClient.kt

```diff
@@ -0,0 +1,283 @@
+/*
+ * Copyright 2008-present MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.mongodb.kotlin.client
+
+import com.mongodb.ClientSessionOptions
+import com.mongodb.ConnectionString
+import com.mongodb.MongoClientSettings
+import com.mongodb.MongoDriverInformation
+import com.mongodb.client.MongoClient as JMongoClient
+import com.mongodb.client.MongoClients as JMongoClients
+import com.mongodb.connection.ClusterDescription
+import com.mongodb.lang.Nullable
+import java.io.Closeable
+import org.bson.Document
+import org.bson.conversions.Bson
+
+/**
+ * A client-side representation of a MongoDB cluster.
+ *
+ * Instances can represent either a standalone MongoDB instance, a replica set, or a sharded cluster. Instance of this
+ * class are responsible for maintaining an up-to-date state of the cluster, and possibly cache resources related to
+ * this, including background threads for monitoring, and connection pools.
+ *
+ * Instances of this class serve as factories for [MongoDatabase] instances. Instances of this class can be created via
+ * the [MongoClient.create] helpers
+ *
+ * @see MongoClient.create
+ */
+public class MongoClient(@PublishedApi internal val wrapped: JMongoClient) : Closeable {
+
+    /**
+     * A factory for [MongoClient] instances.
+     *
+     * @see MongoClient
+     * @since 4.10
```

Comment:
Note: this is the only place I noticed this javadoc annotation

Reply:
Thanks - missed a few